tests/foreman/ui/test_oscaptailoringfile.py

```diff
@@ -27,11 +28,24 @@
     stubbed,
     upgrade,
 )
+from robottelo import ssh
 
 
 @fixture(scope='module')
 def oscap_tailoring_path():
-    return settings.oscap.tailoring_path
+    # download tailoring file from satellite
```

Comment:
@san7ket @ntkathole any idea how I can directly download file from url in this case.(on m/c from where tests are run)

Reply:
Added robottelo/download_file.py to solve this problem.

Reply:
@jameerpathan111 Can't you use / modify [`ServerFileDownloader` from `robottelo/helpers.py`](https://github.com/SatelliteQE/robottelo/blob/master/robottelo/helpers.py#L53)?

Also, in Airgun we have [`get_downloads_list` and `save_downloaded_file`](https://github.com/SatelliteQE/airgun/blob/master/airgun/browser.py), which might be useful. But they are not exposed to Robottelo, you can use them only from within Airgun.

Reply:
thanks @mirzal , I will look into it.

Reply:
@mirzal @jyejare  [ServerFileDownloader from robottelo/helpers.py](https://github.com/SatelliteQE/robottelo/blob/master/robottelo/helpers.py#L53) doesn't provide user with much options, it's not even used anywhere therefore instead of modifying it I think the new function `file_downloader` is much more better. 'file_downloader' seems to satisfy most of requirements.
(I am using wget instead of curl because it helps in case url provided is invalid.)
Should I remove `ServerFileDownloader`? What do you think?

Reply:
@jameerpathan111 if it doesn't meet your requirements and since it's not used anywhere, I vote for removing it and providing better replacement.

Reply:
@mirzal I have added `def file_downloader` in `robottelo/helpers.py` as replacement. Should I raise new pr to remove  `ServerFileDownloader`? 
But @jyejare was about to use `ServerFileDownloader`  because this pr is not merged yet.